state/oracledatabase/oracledatabase.go

```diff
@@ -48,16 +48,16 @@ func newOracleDatabaseStateStore(logger logger.Logger, dba dbAccess) *OracleData
 }
 
 // Init initializes the SQL server state store.
-func (o *OracleDatabase) Init(metadata state.Metadata) error {
-	return o.dbaccess.Init(metadata)
+func (o *OracleDatabase) Init(ctx context.Context, metadata state.Metadata) error {
+	return o.dbaccess.Init(ctx, metadata)
 }
 
 func (o *OracleDatabase) Ping() error {
```

Comment:
Make sure you update all `Ping() error` since you updated the health.Pinger interface